install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -32,18 +32,25 @@ $ openstack role add --user <username> --project <project> _member_
 
 The default quotas assigned to new RH OSP projects are not high enough for
 {product-title} installations. Increase the quotas to at least 30 security
-groups, 200 security group rules, and 200 ports. If kuryr SDN is enabled, and
-particularly if namespace isolation is used, increase the quotas to at least
-100 security groups, 500 security group rules, and 500 ports.
+groups, 200 security group rules, and 200 ports.
 ----
 $ openstack quota set --secgroups 30 --secgroup-rules 200 --ports 200 *<project>*
 ----
 
-If Kuryr-SDN is used, with the namespace isolation feature enabled, there is
-an extra step to make once the project is created. It needs to be added to
-octavia.conf configuration to ensure LoadBalancer security groups belong to
-that project and can be updated to enforce services isolation across
-namespaces.
+[[osp_accounts_kuryr]]
+==== Extra steps for Kuryr SDN
+
+If kuryr SDN is enabled, and particularly if namespace isolation is used,
+increase the quotas to at least 100 security groups, 500 security group
+rules, and 500 ports.
+----
+$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 *<project>*
```

Comment:
Remove the asterisks around \<project\>. They won't render right. You can add an annotation to the line if you want to emphasize the fact that you need to specify the project, like this:

```
----
$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 <project> <1>
----
<1> For `<project>`, specify the name of the project to modify
```